homeassistant/components/hue/event.py

```diff
@@ -89,7 +90,8 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
     @property
     def name(self) -> str:
         """Return name for the entity."""
-        return f"{super().name} {self.resource.metadata.control_id}"
+        # this can be translated too as soon as we support arguments into translations ?
+        return f"Button {self.resource.metadata.control_id}"
```

Comment:
What was `super().name`?

Reply:
the old way of naming it in the entity base model which was removed